### ISSUE TITLE
pharmasig abstract submission

### ISSUE THREAD

**Issue by lengning** [RConsortium/submissions-wg#96]
Hi @bms63 , thanks for agreeing to submit the abstract for pharmasig. please see below a previous abstract that you may find useful

In recent years, statisticians and analysts from both industry and regulatory agencies have increased adoption of open-source software such as R. Compared to other statistical programming languages, R brought great benefit from its vibrant open-source community, providing a wealth of cutting-edge statistical tools and adaptability to the latest data science trends. Particularly, R shiny provided great flexibility and interactivity to increase data scientists’s productivity.  To test the concept that R language and shiny app can be bundled into a submission package and transferred successfully to FDA reviewers, the R consortium R submissions working group successfully completed two pilot submissions and received FDA CDER response letters ([pilot 1,](https://www.r-consortium.org/blog/2021/12/08/successful-r-based-test-package-submitted-to-fda) [pilot 2](https://www.r-consortium.org/announcement/2023/10/05/shiny-app-successfully-reviewed-by-fda-cder-staff-pilot-2-announcement-2)). To our knowledge, these are the first publicly available submission packages that include components of open-source languages. In this talk, I will introduce The R consortium R submission Working Group and the completed pilot 1 and 2

**Comment by bms63** [comment]
Awesome thanks @lengning.  I have to submit internally to GSK to make sure I'm allowed to represent.  I think it should be fine.  Will let you know ASAP.

**Comment by bms63** [comment]
@lengning I submitted abstract today.  Hopefully know if I can present by the next WG meeting.

**Comment by lengning** [comment]
Hi @bms63 would you mind share the final abstract you used for submission? we may use it for UseR submission as well

**Comment by bms63** [comment]
Hi Neng,

Here is what I submitted to Pharmsug-2024

In recent years, statisticians and analysts from both industry and regulatory agencies have increased adoption of open-source software such as R. R brings great benefits from its vibrant open-source community by providing a wealth of cutting-edge statistical tools, extension packages for interactive dashboard and documentation as well as adaptability to the latest data science trends. Particularly, the dashboard focused R package Shiny has shown to provide great flexibility and interactivity.  However, publicly available drug submissions with the R language as the core analysis language has been lacking and limits wider adoption within the Pharma industry.  The R Consortium R Submission Work Group seeks to test the concept that a R-based language submission language can be bundled into a submission package and transferred successfully to FDA reviewers. As of May 2024, the R consortium R submissions working group has successfully completed three pilot submissions and received FDA CDER response letters. To our knowledge, these are the first publicly available submission packages that include components of open-source languages. In this talk, I will introduce the R consortium R submission Working Group and the completed Pilot 1, 2 and 3 findings, issues that we encountered, learnings as well as current work being done in Pilot 4.
